### DISCUSSION TITLE
Welcome to contextualSpellCheck Discussions!

### DISCUSSION THREAD

**Discussion by R1j1t** [R1j1t/contextualSpellCheck#42] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dragonnikkirocks** [comment]
Hii!
Really cool project. I would like to make a german version of spell checker. Do you have any advice?
Thanks a lot in advance

**Reply by R1j1t** [reply]
Hi @dragonnikkirocks happy you like the project!

regarding the question you asked I recently answered a similar question. This is the link to my comment: https://github.com/R1j1t/contextualSpellCheck/issues/44#issuecomment-757339592

I hope it will work for you!
